src/cmd/bootstrap.test.ts

```diff
@@ -80,7 +80,7 @@ describe('Bootstrapping values', () => {
       }),
     )
   })
-  describe('Generating a loose schema', () => {
+  xdescribe('Generating a loose schema', () => {
```

Comment:
what is  xdescribe for?

Reply:
expected as we could still use the method to surgically remove unwanted structures and so we have the skeleton.